tensorbay/client/tests/test_JobMixin.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python3
```

Comment:
The file name does not conform to our specification. `test_version.py` will be better.